src/demo/demo.js

```diff
@@ -87,7 +90,15 @@ class SDK extends Component{
   }
 
   render () {
-    return <div id="onfido-mount"></div>
+    () => { throw new Error('This should NOT be tracked')}
+    return (
+      <div>
+        <div id="onfido-mount"></div>
+        <p>Click this element to trigger an exception:</p>
+        <button onClick={() => { throw new Error('This error should not be tracked') }}>With no class</button>
+        <button class="onfido-sdk-ui" onClick={() => { throw new Error('This error should not be tracked either')}}>With .onfido-sdk-ui class</button>
```

Comment:
This *should* be tracked, right?

Reply:
This should not be tracked even if it has the right class because it's not part of the onfido.min.js or any other onfido file. This element belongs to the host app and its breadcrumbs should not be tracked regardless of its class.

Reply:
Ahh, I see. Thanks!